include/HUDL/HUDL.hpp

```diff
@@ -30,37 +38,240 @@ class HUDL {
     void dataWrite(uint8_t data);
 
     /**
-    * Writes commands to the LCD to control the ST7565
-    *
-    * @param data being written for the command
-    */
+     * Writes commands to the LCD to control the ST7565
+     *
+     * @param data being written for the command
+     */
     void commWrite(uint8_t data);
 
     /**
-    * Writes data to a single pixel
-    *
-    * @param[in] page is the page address to write data to
-    * @param[in] colUp is the first four bits of the column write
-    * @param[in] colLow is the last four bits of the column write
-    * @param[in] data is the data value to write
-    */
+     * Writes data to a single pixel
+     *
+     * @param[in] page is the page address to write data to
+     * @param[in] colUp is the first four bits of the column write
+     * @param[in] colLow is the last four bits of the column write
+     * @param[in] data is the data value to write
+     */
     void drivePixel(uint8_t page, uint8_t colUp, uint8_t colLow, uint8_t data);
 
     /**
-    * Clears the screen
-    *
-    * @param[in] lcd_string
-    */
-    void clearLCD(const uint8_t* lcd_string);
+     * Clears the screen
+     *
+     * @param[in] bitMap a pointer to the bitmap to be displayed
+     */
+    void clearLCD(const uint8_t* bitMap);
 
     /**
      * Initializes LCD for use
      */
     void initLCD();
 
+    /**
+     * Gets the object dictionary
+     *
+     * @return an object dictionary
+     */
+    CO_OBJ_T* getObjectDictionary();
+
+    /**
+     * Gets the size of the Object Dictionary
+     *
+     * @return uint16_t size of the Object Dictionary
+     */
+    uint16_t getObjectDictionarySize() const;
+
+    void displayMap(uint8_t* bitmap);
+
+    /**
+     * Gets BMS Voltage values
+     *
+     * @return uint16_t* pointer to voltage values
+     */
+    uint16_t* getVoltages();
+
+    /**
+     * Gets temperature values
+     *
+     * @return uint32_t* pointer of temperature values
+     */
+    uint32_t* getThermTemps();
+
 private:
+    /**
+     * reg_select PA_3
+     * reset      PB_3
+     * cs         PB_12
+     */
     DEV::LCD lcd;
+    uint16_t voltages[1] = {};
```

Comment:
I don't think you need to track the voltage data because that's not going to be useful to display on the LCD

Reply:
This was something LJ requested. In the next PR this is just a total voltage and not an array of voltages.